src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -7832,6 +7832,32 @@
               # Targeting Spring 2024 Matter release
               - TimeSnapshot
               - TimeSnapshotResponse
+          NetworkCommissioning:
+              # Targeting Spring 2024 Matter release
+              - QueryIdentity
+              - QueryIdentityResponse
+      command fields:
+          NetworkCommissioning:
+              # Targeting Spring 2024 Matter release
+              AddOrUpdateWiFiNetwork:
+                  - networkIdentity
+                  - clientIdentifier
+                  - possessionNonce
+              NetworkConfigResponse:
+                  - clientIdentity
+                  - possessionSignature
+              QueryIdentity:
+                  - keyIdentifier
+                  - possessionNonce
+              QueryIdentityResponse:
+                  - identity
+                  - possessionSignature
```

Comment:
```suggestion
```
Not needed: the commands are provisional, and our setup is smart enough to assume that means all the fields for them are provisional.

Reply:
I thought they'd be needed when we move it across to "introduced" later?